scripts/build_triptych_config.ts

```diff
@@ -0,0 +1,275 @@
+/**
+ * This script provides some type checking and "spell-checking" on triptych config entries, and lets you skip duplicating all of the common bits of JSON.
+ * To use, go to https://www.typescriptlang.org/play and paste this whole thing in, then hit "Run".
+ * You may first need to click "TS Config"  and set Target to ES2019 or later.
+ *
+ * The resulting JSON will be printed to the console. It will be an object with two top-level fields:
+ * - `screenConfig`: the main configuration for all of the triptychs, to be merged into the contents of screens-(dev|dev-green|prod).json
+ * - `playerNameMapping`: the mapping from player name to screen ID. This should be dropped into triptych-player-(dev|dev-green|prod).json
+ *
+ * To change the common parts of the config objects, e.g. to make train crowding widget enabled by default or add a new PSA set, edit the body of `makeScreenConfig`.
+ */
+
+/**
+ * 👉 IMPORTANT NOTES
+ * ------------------
+ * - Screen IDs are of the form `TRI-${StationName}-${routeID}-${directionID}-${index}`
+ * - The last part, the `index`, starts at 1 for the triptych *nearest the front of the train*, and increments from there.
```

Comment:
FWIW I also find left-to-right easier and originally planned to do it that way. I ended up going with this index-from-front-of-train approach because Ana preferred it, and she's going to be working with this stuff more than us—monitoring splunk logs and occasionally editing config.